ModelViewer/ModelViewer/ViewerGraphicsWindow.h

```diff
@@ -29,4 +48,14 @@ class ViewerGraphicsWindow : public OpenGLWindow
     int m_frame = 0;
 
     Model m_currentModel;
-};
\ No newline at end of file
+
+    // Mouse functions
+    void mousePressEvent(QMouseEvent*) override;
```

Comment:
I think these override functions should be under `protected:`

https://doc.qt.io/qt-5/qwindow.html#protected-functions

Reply:
Good call.